components/Send/Send.tsx

```diff
@@ -88,15 +89,17 @@ const Send: React.FunctionComponent<SendProps> = ({
   const { decimalSeparator } = getNumberFormatSettings();
   // transparent is not spendable.
   const spendable = totalBalance.spendablePrivate + totalBalance.spendableOrchard;
-  const stillConfirming = parseFloat(spendable.toFixed(8)) !== totalBalance.total - totalBalance.transparentBal;
+
+  const stillConfirming =
+    parseFloat(spendable.toFixed(8)) !== totalBalance.total - (someUnconfirmed ? 0 : totalBalance.transparentBal);
   const showShieldInfo =
     totalBalance &&
-    totalBalance.transparentBal > 0 &&
-    totalBalance.transparentBal + totalBalance.privateBal > info.defaultFee;
+    (someUnconfirmed ? 0 : totalBalance.transparentBal) > 0 &&
```

Comment:
i see this a lot where the transparent balance is set to 0 when notes are confirming.
would it be possible to implement a spendableTransparent balance in the future?
i will ask in chat as i think this should be done in zingolib?

Reply:
oh, Yes.... that would be perfect... I have to do all this stuff because I haven't that field...